egs/uw3/v1/local/chain/compare_wer.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# this script is used for comparing decoding results between systems.
+# e.g. local/chain/compare_wer.sh exp/chain/cnn{1a,1b}
+
+# ./local/chain/compare_wer.sh exp_pad/chain/cnn1b/ exp_pad/chain/cnn1f/
+# System                          cnn1b     cnn1f
+# WER                              7.68      5.03
+# CER                              2.00      0.45
+# Final train prob              -0.0286   -0.0368
+# Final valid prob              -0.0704   -0.0398
+# Final train prob (xent)       -0.2737   -0.0879
+# Final valid prob (xent)       -0.4723   -0.0991
+
+# exp_pad/chain/cnn1f/: num-iters=150 nj=3..10 num-params=3.6M dim=40->273 combine=-0.04->-0.04
```

Comment:
You don't need to append any actual outputs to this file.
@aarora8, please remove them from your PR if they are there.